src/styles/main.scss

```diff
@@ -0,0 +1,42 @@
+@import './utils/mixin';
+@import './utils/variables';
+
+.page {
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: ($cellSize * $weakSize) +6px;
+  padding-inline: $cellPading;
+
+  &__day {
+    width: $cellSize;
+    height: $cellSize;
+    background-color: $cellCollor;
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    border: solid 1px black;
+
+    font-family: $cellFontFamily;
+    font-size: $cellFontHeight;
+
+    &:hover {
+      cursor: pointer;
+      transform: translateY($translateY);
+      transition: transform $transformDuration;
+    }
+  }
+
+  @include number-day;
+  @include weak-day-lentgh;
```

Comment:
The mixin `weak-day-lentgh` seems to have a typo. It should be `week-day-length` if it's meant to adjust the length of days in a week.